cordova-lib/src/plugman/platforms/ios.js

```diff
@@ -29,8 +29,8 @@ var path = require('path')
   , plist = require('plist')
   , shell = require('shelljs')
   , events = require('../../events')
-  , cachedProjectFiles = {}
-  ;
+  , _ = require('underscore')
+  , cachedProjectFiles = {};
```

Comment:
You can follow the current coding convention and put the semicolon on a new line :smile: